File: Daily Flash/week 4/day 2/MySolutions/Python/Program4.py
print("\n     ------  Pattern -------    \n\n")

num=7

for i in reversed(range(4)):
	for j in range(0,i+1):
		print(num,end="   ")
		num=num+7
	print("\n")
	

print()		

'''
(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 2/MySolutions/Python$ python3 Program4.py

     ------  Pattern -------    


7   14   21   28   

35   42   49   

56   63   

70   





'''

File: Daily Flash/week 4/day 2/MySolutions/C/Program4.c
/*
Problem Statement :

Program 4: Write a Program to Print following Pattern.

7   14  21  28
35  42  49
56  63
70

*/

#include<stdio.h>

void main() {

	int num=7;
	printf("\n    ---------- Pattern --------     \n\n");
	
	for(int i=4;i>0;i--) {
		
		for(int j=0;j<i;j++) {
			
			printf("   %-2d",num);
			num=num+7;
		}
		printf("\n\n");
	}
}


/*
Output : 

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 2/MySolutions/C$ cc Program4.c
(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 2/MySolutions/C$ ./a.out

    ---------- Pattern --------     

   7    14   21   28

   35   42   49

   56   63

   70



*/
